obspy/clients/iris/tests/test_client.py

```diff
@@ -166,7 +166,7 @@ def test_evalresp(self):
                             filename=tempfile)
             with open(tempfile, 'rt') as fp:
                 self.assertEqual(fp.readline(),
-                                 '1.000000E-05  1.055999E+04  1.792007E+02\n')
+                                 '1.000000E-05 1.055999E+04 1.792007E+02\n')
```

Comment:
if these keep making problems you could also `.split()` and compare as a list

Reply:
It looks like it's going to pass so I would leave it as it is.

Reply:
It looks like it should be okay, but yeah, that would be a better approach.

Reply:
I don't care.. if it's passes for now I'm happy